docs/Plugins/webhook.md

```diff
@@ -100,27 +94,39 @@ Read more in Swagger: http://localhost:4000/api/swagger/index.html#/plugins%2Fwe
 
 ## Deployments
 
-Adding pipeline webhook in your deploy shells will help you collect data into DevLake.
+If your CI/CD tool isn't already supported by DevLake, you can insert curl commands in your CI/CD script to post deployment data to DevLake.
+
+DevLake models CI/CD domain with two entities: Pipelines and Tasks. A pipeline has many tasks.
```

Comment:
Semantics/Wording fix:

DevLake CI/CD domain-models contain two entities: Pipelines and Tasks. A pipeline can have one or more tasks.